response_operations_ui/views/reporting_units.py

```diff
@@ -57,7 +58,12 @@ def view_reporting_unit(ru_ref):
         info_message = f'Response status for {survey["surveyRef"]} {survey["shortName"]}' \
                        f' period {period_arg} changed to {new_status}'
 
+    info_arg = request.args.get('info')
+    if info_arg:
+        info_message = 'Verification email re-sent'
```

Comment:
Missing test coverage for message (or is this something to be checked in https://github.com/ONSdigital/rasrm-acceptance-tests/)?